### ISSUE TITLE
Error pulling dependency in CB 0.8.12

### ISSUE THREAD

**Issue by xj9** [ChicagoBoss/ChicagoBoss#555]
I'm seeing this error when running `./rebar get-deps`

```
Pulling uuid from {git,"git://gitorious.org/avtobiff/erlang-uuid.git",
                       {tag,"9cfe9666f1"}}
Cloning into 'uuid'...
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
ERROR: git clone -n git://gitorious.org/avtobiff/erlang-uuid.git uuid failed with error: 128 and output:
Cloning into 'uuid'...
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by danikp** [comment]
use latest HEAD of boss_db to solve it

**Comment by aherranz** [comment]
Same problem in 0.8.14. I changed gitorious.org (that it is going to be shut down) in the deps/boss_db/rebar.config to github.com in my local developments.

Easy to integrate in 0.8.15 ;)

Regards.

**Comment by danikp** [comment]
fixed in v0.8.15
